src/components/propositionalLogic/examplePage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+
+import styles from "./propositionalLogic.module.scss";
+
+class ExamplesPropositionalLogic extends React.Component {
+	public render() {
+		return <div className={styles.site}>
+			<div className={styles.proofEditor}>
+				<p><button className={styles.buttons}
+					onClick={this.showElement}>
+					Beispiel 1
+						</button></p>
+				<table className={styles.toggleTable} id="toggleTable">
+					<tr>
+						<th>Eingabe</th>
+						<th>Erklärung (mit der Maus drüberfahren zum anzeigen)</th>
+					</tr>
+					<tr>
+						<th>Wir zeigen: a -> (b -> c) -> (a und b) -> c.</th>
+						<th>Dies soll bewiesen werden. </th>
+					</tr>
+					<tr>
+						<td></td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>Es seien a, b und c Aussagen.</td>
```

Comment:
This can be rendered more dynamically. In PR #92 there is a dynamical rendering of these so all the tr and td won't need to be retyped and copied etc.